icu4c/source/i18n/unicode/measunit.h

```diff
@@ -72,159 +72,270 @@ enum UMeasureUnitComplexity {
 };
 
 /**
- * Enumeration for SI prefixes, such as "kilo".
+ * Enumeration for SI and binary prefixes, e.g. "kilo-", "nano-", "mebi".
  *
- * @draft ICU 67
+ * Enum values should be treated as opaque: use umeas_getPrefixPower() and
+ * umeas_getPrefixBase() to find their corresponding values.
+ *
+ * @draft ICU 69
  */
-typedef enum UMeasureSIPrefix {
+typedef enum UMeasurePrefix {
+    /**
+     * The absence of an SI or binary prefix.
+     *
+     * @draft ICU 69
+     */
+    UMEASURE_PREFIX_ONE = 30 + 0, // An arbitrary value
 
     /**
      * SI prefix: yotta, 10^24.
      *
-     * @draft ICU 67
+     * @draft ICU 69
      */
-    UMEASURE_SI_PREFIX_YOTTA = 24,
+    UMEASURE_PREFIX_YOTTA = UMEASURE_PREFIX_ONE + 24,
```

Comment:
as suggested use `SI_PREFIX_OFFSET` instead of `UMEASURE_PREFIX_ONE`

Reply:
UMEASURE_PREFIX_ is the symbol prefix that indicates this is an enum value of the UMeasurePrefix type. This is the style of the public ICU4C APIs - partially also because C does not have namespaces, so public symbols need to be very clear. Again: this is the public API that was approved by the committee.

Reply:
See above for comment about `UPRV_SI_PREFIX_OFFSET`